pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/QueryRunnerTestBase.java

```diff
@@ -205,4 +253,11 @@ private static List<String> toH2FieldNamesAndTypes(org.apache.pinot.spi.data.Sch
     }
     return fieldNamesAndTypes;
   }
+
+  public static class TestCase {
+    public String _sql;
+    public String _name;
+    public Map<String, List<String>> _tableSchemas;
+    public Map<String, List<Object[]>> _tableInputs;
+  }
```

Comment:
created a test case record class. later we can use markup language to compose these.